smac/smbo/smbo.py

```diff
@@ -126,12 +131,15 @@ def run(self):
 
             self.logger.debug("Intensify")
 
-            self.incumbent, inc_perf = self.intensifier.intensify(
-                challengers=challengers,
-                incumbent=self.incumbent,
-                run_history=self.runhistory,
-                aggregate_func=self.aggregate_func,
-                time_bound=max(0.01, time_spend))
+            try: self.incumbent, inc_perf = self.intensifier.intensify(
+                  challengers=challengers,
+                  incumbent=self.incumbent,
+                  run_history=self.runhistory,
+                  aggregate_func=self.aggregate_func,
+                  time_bound=max(0.01, time_spend))
+            except TAEAbortException:
```

Comment:
What would be the alternative? Issuing a failed SMAC-run/ raising an exception? I used 
>  The SMAC object should always return an incumbent. (Issue/#155)

as a guideline.